### PR TITLE
Fix  Alerts count according to country filter 

### PR DESCRIPTION
**Pull request by roshni73**
### Address
   https://github.com/IFRCGo/alert-hub-web-app/issues/84

## Changes
- Fix all Ongoing alerts of map according to  country filter  same as table alerts count

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: package.json
Comment (samshara) [inline package.json:1]: Why was this file changed? Let's undo the changes. If we had added any packages. Let's run `yarn add @packageName` to add it.

File: src/views/Home/AlertsView/index.tsx [hunk 1]
```diff
@@ -43,10 +52,17 @@ query CountryList($alertFilters: AlertFilter) {
...
+    public{
+      alerts(filters: $filters) {
         count
+        items {
+            country {
+                id
+                name
+                alertCount
+            }
+        }
```
Comment (samshara) [inline src/views/Home/AlertsView/index.tsx]: Let's add `id` to the `public` node. Apollo graphql uses `normalized` caching. Please go through this https://www.apollographql.com/docs/react/caching/overview/ on how it is done and why the cache id is required.  @barshathakuri @roshni73

File: src/views/Home/AlertsView/index.tsx [hunk 2]
```diff
@@ -74,10 +116,22 @@ function AlertsView(props: Props) {
...
     );
 
+    const variables = useMemo<{ filters: AlertFilter}>(() => ({
+
+        filters: filter,
+    }), [
+
+        filter,
+    ]);
+
```
Comment (samshara) [inline src/views/Home/AlertsView/index.tsx]:
```suggestion
    const variables = useMemo<{ filters: AlertFilter}>(() => ({
        filters: filter,
    }), [

        filter,
    ]);

```

**Review by samshara — DISMISSED** [review]

**Review by barshathakuri — APPROVED** [review]
